doc/guide/guide-basics.rst

```diff
@@ -152,6 +152,8 @@ Therefore, QuTiP includes predefined objects for a variety of states and operato
 | (for n particles)        |                                  |                                        |
 +--------------------------+----------------------------------+----------------------------------------+
 
+and operators:
```

Comment:
This repeats the `and operators` from the sentence above the tables. I agree that having the two tables immediately after each other looks a bit odd, but the headings of the tables do make it clear what is listed in each.